.github/workflows/backport.yml

```
@@ -26,7 +26,7 @@ jobs:
           fetch-depth: 0
           ref: ${{ github.event.pull_request.head.sha }}
       - name: Create backport PRs
-        uses: zeebe-io/backport-action@v0.0.8
+        uses: zeebe-io/backport-action@v0.0.9
```

```suggestion
        uses: zeebe-io/backport-action@2ee900dc92632adf994f8e437b6d16840fd61f58 # tag=v0.0.9
```

How do you feel about pinning the action to the specific commit?

This could help in the situation where the upstream repository of the action is compromised is some ways (especially since the job has some write permissions).

---

It looks like this has already been discussed in #140301, sorry for the noise and feel free to disregard my comment. Dependabot support has however been improved since then and updating the commit reference and the associated comment is not an issue anymore (see https://github.com/dependabot/dependabot-core/pull/5951)